### PR TITLE
[DS-228]: feat: changes step in color util

### PR DESCRIPTION
**Pull request by michalisKout**
## Description 

- changes step in color util from 10% to 5% after design team request.

[jira task](https://orfium.atlassian.net/browse/DS-228)

### CODE REVIEW

**Review by elektracodes — APPROVED** [review]

**Comment by panvourtsis** [comment]
shall i recommend put in a ticket for this?

File: src/components/storyUtils/ColorUtility/useColors.ts [hunk 1]
```diff
@@ -4,16 +4,17 @@ import { validateColor } from './utils';
...
   const [color, setColor] = useState(defaultColor || DEFAULT_COLOR);
   useEffect(() => {
     if (defaultColor !== DEFAULT_COLOR) {
       setColor(defaultColor || DEFAULT_COLOR);
     }
   }, [defaultColor]);
 
-  const [lightenValue, setLightenValue] = useState(0.1);
-  const [darkenValue, setDarkenValue] = useState(0.1);
+  console.log(step);
```
Comment (panvourtsis) [inline src/components/storyUtils/ColorUtility/useColors.ts:15]: remove console log

**Comment by michalisKout** [comment]
> shall i recommend put in a ticket for this?

This would help a lot. Although even if we create a ticket, the design team has to define the acceptance criteria for this tool.

**Comment by michalisKout** [comment]
@panvourtsis based on the last conversation we had in DS refinement, can we merge this to master?

**Comment by panvourtsis** [comment]
> @panvourtsis based on the last conversation we had in DS refinement, can we merge this to master (without opening a task)?

that the reason. We want the criteria in order to see what is changing, breaking etc

**Review by panvourtsis — CHANGES_REQUESTED** [review]
On our latest talk with design team we said that we will need to do a research around the tool they are using to avoid the inconsistencies from our tool. Check ticket for more

**Comment by michalisKout** [comment]
> On our latest talk with design team we said that we will need to do a research around the tool they are using to avoid the inconsistencies from our tool. Check ticket for more

I checked your comment in Jira and I agree that we should investigate it more. 
Although I believe that this PR should be merged to master since this color utility already exists in the production version of Ictinus. 
This is just a fix to help designers work on their new color palette.

**Comment by panvourtsis** [comment]
I am closing this as the ticket will change to a research around the tool that the design team is using

**Pull request closed without merging**